Nimiq.d.ts

```diff
@@ -41,7 +41,35 @@ declare namespace Nimiq {
         static toHex(buf: Uint8Array): string
     }
 
-    class SerialBuffer extends Uint8Array {}
+    class SerialBuffer extends Uint8Array {
+        constructor(bufferOrArrayOrLength: any)
+        subarray(start: number, end: number): Uint8Array
+        readPos: number
+        writePos: number
+        reset(): void
+        read(length: number): Uint8Array
```

Comment:
Unint8Array => SerialBuffer?

Reply:
No, the core is returning a `Uint8Array` here.